directory-connector/src/com/untangle/app/directory_connector/ActiveDirectoryManagerImpl.java

```diff
@@ -201,6 +199,9 @@ public List<GroupEntry> getGroupEntries( String domain, boolean fetchMemberOf )
             if(!adAdapter.getSettings().getEnabled()){
                 continue;
             }
+            if(domain == "" || adAdapter.getSettings().getLDAPHost() == "" || adAdapter.getSettings().getLDAPHost() == null){
```

Comment:
Use `StringUtils.EMPTY` constant for empty string `""`.
Use `StringUtils.isEmpty(adAdapter.getSettings().getLDAPHost())` for null and empty check.